src/tools/miri/tests/pass/intrinsics/portable-simd.rs

```diff
@@ -1,10 +1,22 @@
 //@compile-flags: -Zmiri-strict-provenance
-#![feature(portable_simd, adt_const_params, core_intrinsics, repr_simd)]
+#![feature(
+    portable_simd,
+    unsized_const_params,
+    adt_const_params,
+    intinsics,
+    core_intrinsics,
```

Comment:
is there supposed to be an `intrinsics` and `core_intrinsics` feature here?

Reply:
`intrinsics` seems to be necessary to define the extern block for `simd_shuffle_generic`.

Reply:
huh.